src/views/Home.jsx

```diff
@@ -1,12 +1,76 @@
 import './Home.css';
 import { SingleList } from '../components/SingleList.jsx';
+import { useState, useEffect } from 'react';
+import { createList } from '../api/firebase.js';
+import { useNavigate } from 'react-router-dom';
+
+export function Home({ data, userId, userEmail, setListPath }) {
+	const [newListName, setNewListName] = useState('');
+	const [newListSuccess, setNewListSuccess] = useState(null);
+	const [message, setMessage] = useState(null);
```

Comment:
praise: I like being able to display differing messages using the same variable for our user, I think we could even take the message / setMessage hook and turn it into a very small component that does the alert? 

future suggestion: We might be able to then use it here and in ManageList when a user adds an item to their list, that could enable us to have uniform style and behavior for that alert without having to duplicate code.

Reply:
I like that proposed suggestion to reduce duplicate code. To follow up from a short discussion during office hours, we could also consider using a React package for notifications when we have time at the end to polish the visual design of our app.

Reply:
I like that suggestion too!